src/Sarif/Comparers/ComparerHelpers.cs

```diff
@@ -0,0 +1,206 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.CodeAnalysis.Sarif.Comparers
+{
+    internal class ComparerHelper
```

Comment:
>ComparerHelper [](http://example.com/codeflow?start=19&length=14)

what is our coverage for this and the other comparers ?
can you add in the description, pls?

Reply:
code coverage for new added code is 90%+, added data pr description.